### ISSUE TITLE
Electric chainsaw lacking proper implementation

### ISSUE THREAD

**Issue by SeanMirrsen** [CleverRaven/Cataclysm-DDA#14609]
Right now the electric chainsaw, as I've found recently - since I found an electric chainsaw - is basically a carbon-copy of the regular chainsaw, down to all the stats and functions (besides a combat variant).

This is wrong for several reasons.

A) An electric chainsaw doesn't need to be started. Its "on" switch is basically a safety, so that you don't accidentally cut something useful off of your (or someone else's) body. Even if it _did_ need to be started before use, as a purely electric appliance it would **not** have a chance to randomly not start. When I got the "you flip the switch but nothing happens" message several times in a row on a freshly loaded electric chainsaw, I first thought there was yet another bug with loading items with charges.

B) An average electric chainsaw is, in several key areas, somewhat different from a gas-powered chainsaw. One, it is slightly heavier, owing to the heavy battery pack that's at least partially a block of lead. Two, it is rather less powerful, and the blade tends to be shorter to have less chance of getting stuck in a large log as battery output decreases. Three, it is much quieter when actually running.

C) Related to A, the only time an electric chainsaw consumes power is when it is being used, so there wouldn't be a constant drain of power when keeping the tool "ready for use".

D) This is rather a difference in the other direction, but I would actually like to see a combat modification for the electric chainsaw as well - even if it has less power, the same sort of retooling as applied to a regular chainsaw, could turn an electric chainsaw into a decent if unwieldy weapon as well.

In short, the electric chainsaw needs these modifications:
1) Replace the on/off mechanic with charges per attack - I assume there are melee weapons that do this already? If not, the mechanic needs to be added. Or just have the on/off take very little time and have no fail chance, _and_ add charges per attack.
2) Make it heavier and less powerful than the regular chainsaw, and remove any and all noise made by it except when attacking or using as a tool. Say, bump the weight to about 7kg, and make it deal on the order of 55-60 cutting damage unmodified.
3) Add a combat variation, acquired the same way as the combat chainsaw. Roughly similar stat improvements over the tool variant, though I'd expect the battery size will not improve (it really doesn't need to).

**Comment by Rivet-the-Zombie** [comment]
>One, it is slightly heavier, owing to the heavy battery pack that's at least partially a block of lead.

Not in my experience. Every electric chainsaw I've handled has been a lot lighter than a gasoline powered model.

All good points otherwise.

**Comment by RadaRadaRada** [comment]
Diamond combat chainsaws when?

**Comment by SeanMirrsen** [comment]
>Not in my experience. Every electric chainsaw I've handled has been a lot lighter than a gasoline powered model.
Just to clarify, you do mean a *cordless* electric chainsaw, right?

And additionally, most cordless models would be lighter due to being very small and weak (such as for, at most, cutting firewood for camping). This is assumed to be the sort of large chainsaw what can fell trees, so it'll have a heavy battery to match the power requirements.

**Comment by Rivet-the-Zombie** [comment]
Yes, I was referring to cordless models. The 19.2v batteries that mine uses only weigh about two pounds each, and while it has a fairly short (twelve inch) bar, it's powerful enough to cut down a decent sized tree and still have power to spare.

**Comment by SeanMirrsen** [comment]
Well, if you say so. I certainly won't object to a *lighter* and quieter chainsaw with renewable fuel. :P

**Comment by Coolthulhu** [comment]
The electric chainsaw should probably consume fuel at a faster rate, though.

100 charges of batteries hold much less power than 100 charges of gasoline, as can be easily seen by burning 100 charges of gasoline to power an alternator.
200 gasoline burned to power a car alternator produces more than 10000 charges of battery charge.

Those rates are a bit insane and the electric chainsaw would be unusable if it tried to replicate them, but the point still stands: gasoline is much more volume-efficient and weight-efficient than the batteries.

**Comment by Rivet-the-Zombie** [comment]
Yeah, if we're going to have it use the standard batteries, then it should probably tear through them really fast.

**Comment by SeanMirrsen** [comment]
I think the current battery load of 400 mostly makes sense (compared to the 100 charges powering a big flashlight), but it'd be using at least 10 charges per melee attack. Maybe even 15-20. So it wouldn't have the same lasting power as a gas-powered chainsaw. At the same time - only eats power when used. So no idling costs, and no need to keep it running as you walk around. And assuming something like the rechargeable battery mod can be clipped to it, would become very much renewable via solar power, unlike the slowly dwindling supplies of gasoline.

**Comment by illi-kun** [comment]
### This issue was closed as it appears inactive.

Reducing open issues to those which are (or will) be actively worked upon helps us focus our efforts. This issue has not been deleted - it still appears in searches and if it contains relevant information you are encouraged to continue to link to it.

### If this issue was a bug
It should be reopened if it can be reproduced in the current build. You can obtain the most recent copy [here](http://dev.narc.ro/cataclysm/jenkins-latest/). Please check there is not a more recent report of this bug before doing so. If no more recent report exists you should continue the discussion in this issue.

### If this was a feature request
If the consensus was that the idea was good you could consider submitting an implementation via a PR. If you want to comment further please do so here as opposed to opening a new issue. Before posting check nobody has already made the same point and consider whether your comments are likely to lead to an implementation. If you have doubts about either consider instead [voting](http://iv.cataclysmdda.com:8111/all) for the issue

### If you want to work on this issue
Then either assign it to yourself or if you are unable to do so claim it via adding a comment. Please don't assign others or make a general request for action.
